File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1066_basico.js
import { question } from "readline-sync";

function main() {
    let count = 1;
    let countPair = 0;
    let countOdd = 0;
    let countPositive = 0;
    let countNegative = 0;

    while (count <= 5) {
        let number = get_number();

        if (number % 2 === 0) {
            countPair++;
        } else {
            countOdd++;
        }

        if (number > 0) {
            countPositive++;
        } else if (number < 0) {
            countNegative++;
        }

        count++;
    }
    print(`${countPair} valor(es) par(es)`);
    print(`${countOdd} valor(es) impar(es)`);
    print(`${countPositive} valor(es) positivo(s)`);
    print(`${countNegative} valor(es) negativo(s)`);

}

function get_number() {
    return parseInt(question());
}

function print(texto) {
    return console.log(texto);
}

main();

File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1066_basico.py
pr = 0; im = 0; ps = 0; ng = 0

for i in range(5):
    x = int(input())
    if (x >+ 0): ps += 1
    elif (x < 0): ng += 1
    if (x % 2 == 0): pr += 1
    else: im += 1
print(f"{pr} valor(es) par(es)")
print(f"{im} valor(es) impar(es)")
print(f"{ps} valor(es) positivo(s)")
print(f"{ng} valor(es) negativo(s)")